### PR TITLE
post meta final polish for Issue #14

### LINKED ISSUE #14: Style post meta / stick to bottom of featured image
Style the post meta (.entry-meta) on featured image on single posts and pages as a bar, smaller text, stuck to bottom of the large post header image.

Something like this but nicer looking: 

Screen Shot 2019-07-29 at 11 55 57 AM

CSS notes:
* You will need to remove the margin left/right on this: .site-header.featured-image .site-featured-image .entry-header
* on .entry-meta, something like this:
```
position: absolute;
bottom: -55px; // But not hardcoded like this - should stick to bottom
left: 0;
width: 100%;
font-size: 0.75rem; 
background-color: var(--color-brandRed); // Colors could come from duotone variables instead
color: white;
padding: 0.75rem;
```
* Hid the icons - not sure if we would want those or not

### PR DESCRIPTION
**Pull request by slambert**
styled with same font, weight, and letterspacing as headers.  Made default color the beige. Brought back the icons (for now, I'm 75% on that decision).

Closes #14

### CODE REVIEW

**Comment by slambert** [comment]
Additional work helps with headers on #31

Next time I'll make branches for each of these. Sorry.

File: css/03-generic.css [hunk 1]
```diff
@@ -93,10 +93,18 @@ h2:before, .entry .entry-title:before {
 }
 
 .entry .entry-content > h2 {
-	margin-left: -1rem;
+	margin-left: -2rem;
```
Comment (laras126) [inline css/03-generic.css]: Was this tested on phones? Need to make sure the hardcoded values are still okay for smaller screens.

File: functions.php [hunk 2]
```diff
@@ -154,6 +154,11 @@ function c4aa_setup_theme_supported_features() {
 			'slug' => 'caa-grey',
 			'color' => '#968c83',
 		),
+		array(
+			'name' => __( 'C4AA Black', 'themeLangDomain' ),
```
Comment (laras126) [inline functions.php]: Update all instances of `themeLangDomain` with `c4aa`

File: css/03-generic.css (hunk 1 above)
Comment (slambert) [reply css/03-generic.css]: Assuming you're referring to the two lines below

	margin-top: 64px;
	margin-bottom: 48px;

Just switched them to _rem_ values

However, the margin-left thing could get a little goofy on mobile. We may want to have those negative values change to zero values when there's no left margin to drift into...

File: functions.php (hunk 2 above)
Comment (slambert) [reply functions.php]: Huh? Watcha mean exactly?

Comment (laras126) [reply functions.php]: @slambert Any place that says `'themeLangDomain' should be 'c4aa' - that 'themeLangDomain' is a placeholder from the tutorial!

File: css/03-generic.css (hunk 1 above)
Comment (laras126) [reply css/03-generic.css]: Rem values will have the same problem - they are still static sizes. You could do some fun calc with `vw` units (vw = viewport width / 1vw = 1% of the width of the viewport). Maybe start with `margin-top: calc( 2vw + 30px )`? Then resize the browser window to see how it responds.

File: functions.php (hunk 2 above)
Comment (slambert) [reply functions.php]: Ah thank you!

File: css/03-generic.css (hunk 1 above)
Comment (slambert) [reply css/03-generic.css]: Oops - I requested your review too early. Will aim to get on this sometime today.

Comment (laras126) [reply css/03-generic.css]: @slambert Okay, send me a message when ready!

Comment (slambert) [reply css/03-generic.css]: It's ready @laras126

File: functions.php (hunk 2 above)
Comment (laras126) [inline functions.php]: @slambert still need to update these! should be:
`'name' => __( 'C4AA Black', 'c4aa' ),`

Comment (slambert) [reply functions.php]: This should be done and in commit 9ea31b1

https://github.com/thec4aa/c4aa-org/commit/9ea31b1bb2f7aa46eed72bed43469fb506d0de4d

??

Comment (laras126) [reply functions.php]: Oh - I guess what the PR is showing is outdated? Meh, merging!

**Pull request merged**